lib/include/dlisio/dlisio.hpp

```diff
@@ -1,29 +1,25 @@
-#ifndef DLISIO_H
-#define DLISIO_H
+#ifndef DLISIO_HPP
```

Comment:
Search this file for "DLIS_". There are couple of things in the comments

There is also one in ext/types.hpp